### PR TITLE
Initial commit for Dutch translations of PokéMesh

### PR DESCRIPTION
**Pull request by Tashla**
Arrays.xml is untouched, added most translations in FAQ and Strings with
a comment for some as these were questionable.

### CODE REVIEW

File: values-nl/strings.xml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    <string name="pokemesh_show_on_map">Op kaart weergeven</string>
+    <string name="pokemesh_show_on_map_summary">Geef deze Pokémon wel of niet weer op de kaart</string>
+    <string name="pokemesh_mesh_pokemon">Notificaties</string>
+    <string name="pokemesh_mesh_pokemon_summary">Geef een notificatie als deze Pokémon in de buurt is</string>
+    <string name="pokemesh_pokemesh_disabled">You need to enable PokéMesh settings before proceed!</string>
+    <string name="pokemesh_mesh_disabled">Notificaties uitgeschakeld</string>
+    <string name="pokemesh_mesh_enabled">Notitificaties ingeschakeld</string>
+    <string name="pokemesh_mesh_finding">Geen nabije Pokémon op het moment!</string>
+    <string name="pokemesh_meshed">Een %s is in de buurt!</string>
+    <string name="pokemesh_multi_meshed" formatted="false">Er zijn %s in de buurt van %s!</string> <!-- Questionable! Not sure what the last parameter is used for! -->
```
Comment (iGio90) [inline values-nl/strings.xml]: %s are strings injected by the code. In this case %s will be replaced by the Pokémon name and count!

File: values-nl/strings.xml [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    <string name="pokemesh_big_multi_meshed">Er zijn meerdere Pokémon in de buurt!</string>
+
+    <string name="pokemesh_support_us">PokéMesh steunen</string>
+    <string name="pokemesh_support_us_summary">Na iedere 30 scans een advertentie op volledig scherm tonen</string>
+    <string name="pokemesh_about">Over</string>
+    <string name="pokemesh_follow_facebook" translatable="false">/PokeMesh</string> <!-- May I suggest making these two also non-translatable? Considered that you'd like to keep a single facebook/twitter page for everything! -->
```
Comment (iGio90) [inline values-nl/strings.xml]: correct suggestion!

File: values-nl/strings.xml [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    <string name="pokemesh_gps_failed">GPS locatie ophalen mislukt</string>
+    <string name="pokemesh_init_pokemesh">Verbinden met PokéMesh....</string>
+    <string name="pokemesh_logging_in">Aanmelden bij PTC</string>
+    <string name="pokemesh_connected">Verbonden! Mesh… Mesh… !</string>
+    <string name="pokemesh_generating_credentials">Aanmaken van nieuwe gegevens</string>
+    <string name="pokemesh_pokemesh_down">PokéMesh ondervindt op dit moment onderhoud. Probeer het later opnieuw!</string>
+    <string name="pokemesh_got_pokemesh_credentials">PokéMesh gegevens aangemaakt!</string>
+    <string name="pokemesh_club_down">Pokémon GO servers zijn momenteel offline of niet bereikbaar. Probeer het later opnieuw!</string>
+    <string name="pokemesh_no_internet">Communicatie met het netwerk mislukt. Probeer het later opnieuw!</string>
+    <string name="pokemesh_got_it">Got it!</string> <!-- Questionable! Not sure where this string is used in the app! -->
```
Comment (iGio90) [inline values-nl/strings.xml]: just a general string that will be mostly used on Alert message (if any)

**Comment by iGio90** [comment]
Replied to all the questions! If you have any other, let me know!
Let me also know when i can merge this ^^ Thanks!

**Comment by Tashla** [comment]
Feel free to merge! Edited most of the questionables and left "Got it!" in as this is a more general approach aswell in the Dutch language.

Also might add as a note: I've sent you an e-mail earlier at pokemeshapp@gmail.com, just checking in whether you've received it or not ;)

**Pull request merged**